app/models/host_info_providers/static_info.rb

```diff
@@ -4,6 +4,7 @@ def host_info
       # Static parameters
       param = {}
 
+      param["name"] = host.name unless host.name.nil?
```

Comment:
Would it make sense to have `name` (or `hostname`) as the short name and also a field `fqdn` with the full name, including the domain?

Reply:
@ekohl thank you for the inputs. Agreed and added the FQDN field as well to this PR.

Reply:
Do we still have the issue, that we can override puppet facts with this? So does it make sense to prefix this with `foreman_hostname`?

Reply:
Yes and no. It depend on how you use it in your manifest.
```puppet
$::hostname # Could be overridden via parameters
$facts['hostname'] # Really a fact
```

Reply:
Currently `name` and `fqdn` both show the full name - the one with FQDN:

![image](https://user-images.githubusercontent.com/10046168/98812251-d2df2d80-242a-11eb-9526-0f873b546c2f.png)

Is it intentional?

Reply:
@ShimShtein yes if a host is registered using a FQDN, then both these values are same. They only differ if `append_domain_name_for_hosts` setting is set to `False` .